src/components/projects/new-project.jsx

```diff
@@ -45,6 +52,11 @@ export default function NewProject({ isOpen, onOpen, onClose }) {
     }
   }
 
+  const validateProjectName = (value) => {
```

Comment:
```suggestion
  const validateProjectName = value => {
```
my opinion is an opinion